src/libraries/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -24,12 +24,12 @@ public sealed partial class Font : MarshalByRefObject, ICloneable, IDisposable,
         private IntPtr _nativeFont;
         private float _fontSize;
         private FontStyle _fontStyle;
-        private FontFamily _fontFamily;
+        private FontFamily _fontFamily = null!;
         private GraphicsUnit _fontUnit;
         private byte _gdiCharSet = SafeNativeMethods.DEFAULT_CHARSET;
         private bool _gdiVerticalFont;
         private string _systemFontName = "";
-        private string _originalFontName;
+        private string _originalFontName = null!;
```

Comment:
This seem to be nullable

Reply:
It will get assigned the `string fontFamily` constructor parameter, which is non-nullable.